config/schema.yaml

```diff
@@ -82,22 +83,58 @@ mosdepth_use_fast_algorithm:
   required: false
   default: true
   help: 'fast algorithm ignores read pair overlaps and CIGAR information'
-mosdepth_window_size:
-  type: 'Integer'
-  required: false
-  default: 500
-  help: 'Window size for mosdepth coverage calculation'
 mosdepth_per_base_output:
   type: 'Bool'
   required: false
   default: true
   help: 'Output per-base coverage'
+mosdepth_window_size:
+  type: 'Integer'
+  required: false
+  default: 500
+  help: 'Window size for mosdepth coverage calculation'
 mosdepth_additional_options:
   type: 'String'
   required: false
   allow_empty: true
   default: ''
   help: 'Additional arguments for mosdepth command'
+mosdepth_quantize_cutoffs:
+  type: 'String'
+  required: false
+  default: '0:1:5:150'
+  help: 'Quantize coverage values into these bins'
+mosdepth_quantize_use_fast_algorithm:
+  type: 'Bool'
+  required: false
+  default: false
+  help: 'Use fast algorithm for quantizing coverage values'
```

Comment:
**question (non-blocking)**: Is there a downside to enabling the use of the fast algorithm by default?

Reply:
Yeah, this is a difficult decision.  Not using fast mode gives "correct" results.  Fast mode ignores paired read overlaps and CIGAR strings (thus indels wrt ref).  Ignoring the paired read overlap is the bigger issue, especially for samples with small insert sizes (wrt read length). This is noted in the README.  The time difference isn't clear as I haven't benchmarked more than a few samples.  It's enough that the `mosdepth` author recommends fast mode for most use-cases (I assume non-small insert cases).   We currently have fast mode `true` by default for the regular `mosdepth` coverage calculation, so we should probably change one or the other to make them consistent.

Reply:
That's fair, we can gather more data as needed and then make a decision